docs/how-to/manage/configure-unprivileged-workloads.rst

```diff
@@ -0,0 +1,42 @@
+.. _configure_unprivileged_workloads:
+
+Configure user workloads to run unprivileged
```

Comment:
suggestion (non-blocking): let's capitalize in a coherent way with the other page titles